.github/workflows/erd.yml

```
@@ -0,0 +1,46 @@
+      - 'prisma/schema.prisma'
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Generate and Build ERD
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)